modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/resources/seqexec-semantic-search.html

```diff
@@ -210,155 +210,268 @@
-                            <input type="text" autocomplete="off">
-                          </div>
-                          <div class="required field">
-                            <label>SOS</label>
-                            <input type="text" autocomplete="off">
-                          </div>
-                          <div class="field">
-                            <label>Image Quality</label>
-                            <div class="ui fluid selection dropdown">
-                              <div class="default text">Select</div>
-                              <i class="dropdown icon"></i>
-                              <input type="hidden" name="iq">
-                              <div class="menu">
-                                <div class="item" data-value="male">IQ20</div>
-                                <div class="item" data-value="female">IQ70</div>
-                                <div class="item" data-value="female">IQ85</div>
-                                <div class="item" data-value="female">Any</div>
-                              </div>
-                            </div>
+      <div class="ui raised segments container">
+        <div class="ui top attached text menu segment">
+          <div class="ui header item">Running Sequences</div>
+        </div>
+
+        <div class="ui bottom attached segment">
+          <div class="ui attached tabular menu">
+            <a class="active item" data-tab="first">GPI (GS-2016A-Q-0-1)</a>
+            <a class="item" data-tab="second">GMOS-S (GS-2016A-Q-5-3)</a>
+            <a class="item negative" data-tab="third"><i class="attention icon"></i> F2 (GS-2016A-Q-4-1)</a>
+          </div>
+          <div class="ui bottom attached active tab segment" data-tab="first">
+            <div class="ui grid">
+              <div class="row">
+                <div class="four wide column tablet computer only">
+                  <div class="ui raised secondary segment">
+                    <h4>Headers</h4>
+                    <div class="ui form">
+                      <div class="required field error">
+                        <label>Observer</label>
+                        <input type="text" autocomplete="off">
+                      </div>
+                      <div class="required field">
+                        <label>SOS</label>
+                        <input type="text" autocomplete="off">
+                      </div>
+                      <div class="field">
+                        <label>Image Quality</label>
+                        <div class="ui fluid selection dropdown">
+                          <div class="default text">Select</div>
+                          <i class="dropdown icon"></i>
+                          <input type="hidden" name="iq">
+                          <div class="menu">
+                            <div class="item" data-value="male">IQ20</div>
+                            <div class="item" data-value="female">IQ70</div>
```

Comment:
Hmm, pardon my ignorance but what is the difference between male and female values?